src/base-element.js

```diff
@@ -82,9 +84,18 @@ import {vsyncFor} from './vsync';
  * before buildCallback and it might be called multiple times including
  * after layoutCallback.
  *
- * The documentInactiveCallback is called when the document becomes
- * inactive. E.g. when the user swipes away from the document or
- * focuses a different tab.
+ * The pauseCallback is called when when the document becomes inactive, e.g.
```

Comment:
Just document? What if the video is swiped away in slides? Will we use the same callback?